docs/Users_Guide/overview.rst

```diff
@@ -528,7 +528,7 @@ METplus Components Python Requirements
       libraries like Plotly
     - 
   * - pyyaml
-    - >=5.3.1
+    - >=6.0
```

Comment:
This is listed as 6.0 in calcpy, plotpy, and dataio nco_requirements.txt and 6.0 in plotpy and dataio requirements.txt, but 5.4.1 in calcpy requirements.txt. Should we update calcpy requirements.txt?